atm.js

```diff
@@ -1,3 +1,74 @@
+//Checking variable
+var checkingInput = $('#checking .input');
+var checkingBalance = $('#checking .balance');
+var checkingDeposit = $('#checking .deposit');;
+var checkingWithdraw = ('#checking .withdraw');
+
+//Savings variable
+var savingsInput = $('#savings .input');
+var savingsBalance = $('#savings .balance');
+var savingsDeposit = $('#savings .deposit');
+var savingsWithdraw = $('#savings .withdraw');
+
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+  //Turn background red
+  //$('body').css("background", "red");
+  //Add a click listener to the **checking account's** "Deposit" button
+  console.log("hola");
+  //User input a dollar amount to deposit money to checking
+
+  //Checking deposit button adds user input
+  $(checkingDeposit).on("click", function(evt) {
+    //Prevent default of submit button from refreshing page
+    evt.preventDefault();
+    console.log("Checking deposit button pressed");
+
+    currentChecking = parseFloat($(checkingBalance).html().split('$')[1]);
+    console.log("Current Checking Amount is a type of: " + typeof(currentChecking));
+
+    //update amount
+    checkingInputVal = parseFloat(checkingInput.val());
+    console.log("Checking Input Value is a type of: " + typeof checkingInputVal);
+    $('#checking .balance').html("$" + (checkingInputVal + currentChecking));
+  });
+
+  $(checkingWithdraw).on("click", function(evt) {
+    //Prevent default of submit button from refreshing page
+    evt.preventDefault();
+    console.log("Checking withdraw button pressed");
+
+    currentChecking = parseFloat($(checkingBalance).html().split('$')[1]);
+    console.log("Current Checking Amount is a type of: " + typeof(currentChecking));
+
+    //update amount
+    checkingInputVal = parseFloat(checkingInput.val());
+    console.log("Checking Input Value is a type of: " + typeof checkingInputVal);
+
+    if(checkingBalance > 0) {
+      $('#checking .balance').html("$" + (currentChecking - checkingInputVal));
+    }
+    else {
+      $('.balance').css("background", "red");
+    }
```

Comment:
I think I know why it's always red:

what is the value of checkingBalance on line 47? Is it ever an integer? You defined it on line 3 but never redefined it. Its always a jQuery object and therefore isn't a number and therefore will never be greater than 0. Instead of `checkingBalance` maybe use `currentChecking - checkingInputVal` better yet, store that value in another variable, like `updatedTotal` or something.